src/controller/org.controller/org.middleware.js

```diff
@@ -8,8 +8,8 @@ const utils = require('../../utils/utils')
 function isOrgRole (val) {
   val.forEach(role => {
     if (!CONSTANTS.ORG_ROLES.includes(role)) {
-      logger.info('Invalid role for organization', role)
-      throw new Error('The organization role \'' + role + '\' does not exist.')
+      logger.info({ uuid: req.ctx.uuid, message: 'Invalid role for organization ' + role })
```

Comment:
since this is just a function, we can avoid logging here. instead, we should let the caller decide if they want to log the statement in their catch clause